static/assets/js/eclipsefdn.adopters.js

```diff
@@ -1,4 +1,6 @@
+
+module.exports.json_adopters =  json_adopters;
```

Comment:
Thanks @Linus-CS for looking into this issue!

We can't update this file directly since it's generated by this script:
https://github.com/EclipseFdn/iot.eclipse.org/blob/master/bin/adopters.sh

The source of the file is here: 
https://github.com/EclipseFdn/solstice-assets/blob/master/js/eclipsefdn.adopters.js

Reply:
Oh, ok thank you for the advice. Can I change the file https://github.com/EclipseFdn/solstice-assets/blob/master/js/eclipsefdn.adopters.js or should I redo it differently?

Reply:
I don't think we want to change that file since we are adding that variable via our shell script with the our install script and we want to make sure that our list of adopters is never empty.

Reply:
So now I am testing if https://github.com/EclipseFdn/iot.eclipse.org/blob/master/static/assets/js/eclipsefdn.adopters.json
is empty